packages/eslint-plugin-rules/README.md

```diff
@@ -0,0 +1,70 @@
+# Eslint Plugin Rules
+
+## About
+
+Place for eslint custom rules for Foreman and plugins
+
+## Usage
+
+### Create an eslintrc file
+```js
+{
+  "plugins": ["@theforeman/rules"],
+  "rules": {
+    "@theforeman/rules/require-ouiaid": "warn",
+  }
+}
+```
+If not specified in the eslintrc file, these components will be checked for an `ouiaId` prop:
```

Comment:
Can you add here why we need an `ouiaId` prop in these components, and an example?

Reply:
`ouiaId` field is used for QE's automation tests.

Reply:
@MariaAga Updated.

Reply:
Thanks